snode-src/duckcoin.go

```diff
@@ -375,6 +382,7 @@ func saveKeyPair(pubkey string, privkey string, pubfile string, privfile string)
 	return nil
 }
 
+// loadKeyPair loads a keypair from a PEM file
```

Comment:
The code already has this info (okay maybe not the PEM part, but that doesn't matter much)
```suggestion
```